Turn/src/Store.cpp

```diff
@@ -31,16 +31,26 @@ void Store::StoreFront(Player* _Player) {
 		_Player->DisplayInventory();
 		cout << endl << " Jeremy's Tools Store" << endl;
 		cout << " \"" << random_greeting() << "\"" << endl << endl;
+		cout << "\""<< "Jeremy's Wealth" <<this->getJeremysCoin()<<"\""<<endl<<endl; //display the coin that Jeremy has. 
 		cout << " ----ITEM----------------COST------ " << endl << endl;
 		cout << " 1) Arrows x 5\t\t " << arrowsCost << endl;
 		cout << " 2) Bomb\t\t " << bombCost << endl;
 		cout << " 3) Potion\t\t " << potionCost << endl;
 		cout << " 4) Whetstone\t\t " << whetstoneCost << endl << endl;
 		cout << " 0) Exit" << endl << endl;
 		cout << " ---------------------------------- " << endl << endl;
-		cout << " What do you want to buy today?" << endl << endl << " ";
+		
+
+		cout << " Are you here to buy or to sell?" << endl << endl << " ";
```

Comment:
I'd put a prompt for what to type here, something like
```
1) Buy
2) Sell
3) Exit
```

Reply:
I'd say `3) Exit` should be 0 to keep consistency with other UI elements.